frontend/server/src/Controllers/Course.php

```diff
@@ -134,6 +138,44 @@ private static function validateCreateAssignment(
             'alias',
             /*$isRequired=*/true
         );
+
+        $addedProblems = [];
+        if (!empty($r['problems'])) {
+          /** @var list<array{alias: string, points: int|null}> */
```

Comment:
```suggestion
          /** @var list<array{alias: string, points?: int}> */
```

en L171 no se está checando si `points` es nulo, sino si está puesto. eso se declara con `?`.

Reply:
Se corrige.